unity/PitayaExample/Assets/Pitaya/Editor/PitayaXcodeProjectUpdater.cs

```diff
@@ -21,8 +26,31 @@ public static void OnPostprocessBuild(BuildTarget buildTarget, string path)
 			// Pitaya should be linked with zlib when on iOS.
 			proj.AddBuildProperty(targetGuid, "OTHER_LDFLAGS", "-lz");
 
+
+			foreach (string file in Directory.GetFiles(GetPathToHide(path))) {
```

Comment:
It might be good to add a comment explaining why we need to hide/delete one of the folders.